engines/director/cast.cpp

```diff
@@ -236,6 +236,8 @@ void Cast::loadArchive() {
 bool Cast::loadConfig() {
 	Common::SeekableReadStreamEndian *stream = nullptr;
 	stream = _castArchive->getMovieResourceIfPresent(MKTAG('V', 'W', 'C', 'F'));
+	if (!stream)
```

Comment:
This is better to be checked against the Director version. E.g. something like if (Director < 6) load VWCF, load DRCF otherwise.

Reply:
Original director checks for both judging from the reversing documentation.